# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/rental-impl/.cargo-checksum.json
# Commit: 16350e08d34f
# Full Hash: 16350e08d34ff06c7f46745c6b160c7a7bda9140
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2019-12-06 04:43:57
# Regressor Bug: 1586855
# File Overlap Count: 6
# Description:
#   bug 1586855 - incorporate CRLite filters into cert_storage r=jcj,kjacobs
#   
#   This patch implements CRLite lookups for TLS server certificate revocation
#   information in telemetry-only mode. It adds a new preference
#   "security.pki.crlite_mode" to control the behavior of this feature. Setting
# ==============================================================================

diff -r 64fd40663930 -r 16350e08d34f third_party/rust/rental-impl/.cargo-checksum.json
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/third_party/rust/rental-impl/.cargo-checksum.json	Thu Dec 05 22:41:53 2019 +0000
@@ -0,0 +1,1 @@
+{"files":{"Cargo.toml":"02a12362b4ac9ac229d1eb74ee1d3cb2706b2057a39165161d8862d7e85ee3ac","src/lib.rs":"622d874509d1cc6f170558faf577a08d792f514eb0112a033b530bcb59c2b82b"},"package":"475e68978dc5b743f2f40d8e0a8fdc83f1c5e78cbf4b8fa5e74e73beebc340de"}
\ No newline at end of file